src/main/java/com/enderio/core/common/BlockEnder.java

```diff
@@ -167,6 +168,20 @@ protected boolean shouldDoWorkThisTick(World world, int x, int y, int z, int int
     }
   }
 
+  @Override
+  public boolean hasComparatorInputOverride() {
+    return IComparatorOutput.class.isAssignableFrom(teClass);
```

Comment:
That's a native, since Java 1.6 as fast as "instanceof", which is faster than a method call. So we might be able to shave off a tiny bit with 2 booleans, but I really doubt it'd be worth it.

However, null checking is worth the effort, that thing throws an NPE. Do they *have* to hide that so deep in the docs?